src/parser/rogue/outlaw/modules/spells/SliceAndDiceUptime.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import Statistic from 'interface/statistics/Statistic';
+import { formatPercentage } from 'common/format';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import SpellLink from 'common/SpellLink';
+
+class SliceAndDiceUptime extends Analyzer {
+  get percentUptime() {
+    return (
+      this.selectedCombatant.getBuffUptime(SPELLS.SLICE_AND_DICE.id) / this.owner.fightDuration
+    );
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.CORE(100)}
+        size="flexible"
+      >
+        <div className="pad">
```

Comment:
is this div  needed? Not sure what that is for.

Reply:
It adds padding to the label so it looks better and in line with the other stat components and their labels.

Those above are Azerite Traits. But I thought I deleted them. Wonder if they were reintroduced when I merged the upstream/shadowlands branch in.

Reply:
k. if they are BFA traits, just get rid of them